modules/cnf-topology-aware-lifecycle-operator-about-cgu-crs.adoc

```diff
@@ -0,0 +1,219 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: CONCEPT
+[id="talo-about-cgu-crs_{context}"]
+= About the ClusterGroupUpgrade CR
+
+TALO builds the remediation plan from the `ClusterGroupUpgrade` CR for a group of clusters. You can define the following specifications:
+
+* Clusters in the group
+* Blocking `ClusterGroupUpgrade` CRs
+* Applicable list of managed policies
+* Number of concurrent updates
+* Applicable canary updates
+* Actions to perform before and after the update
+* Update timing
+
+The `ClusterGroupUpgrade` CR can be in the following states:
+
+* `UpgradeNotStarted`
```

Comment:
So this seems to lack UpgradeCannotStart, which is referenced later in the doc, along with the Precaching states.
This should be rethinked on 4.11, UpgradeCannotStart does not warrant IMHO its own state, it could fall within UpgradeNotStarted with just a different message. Or maybe rename it to UpgradeBlocked

Reply:
@rcarrillocruz Is UpgradeCannotStart only applicable when using blocking CRs? Are there other scenarios where UpgradeCannotStart would be applicable?

(For now, I added a note that I should add an entry about this state)

Reply:
@amolnar-rh Yes, UpgradeCannotStart is only applicable for blocking CRs:
- if blocking CRs are missing from the system
- if blocking CRs have not yet completed